istio/Makefile

```diff
@@ -37,6 +37,17 @@ init1: ## Install SSL certs and Istio profile
 init2: ## Install custom manifests
 	kustomize build overlays-2/$(overlay) | kubectl apply -f -
 
+.PHONY: restart_proxies
+restart_proxies: ## Restarts all istio dataplane proxies, can be used when rolling out upgrade
+	kubectl rollout restart deployment/argocd-application-controller -n argocd
+	kubectl rollout restart deployment/argocd-dex-server -n argocd
+	kubectl rollout restart deployment/argocd-redis -n argocd
+	kubectl rollout restart deployment/argocd-repo-server -n argocd
+	kubectl rollout restart deployment/argocd-server -n argocd
+	kubectl rollout restart deployment/doc-index-updater -n doc-index-updater
+	kubectl rollout restart deployment/medicines-api -n medicines-api
+	cd ../observability/prometheus && make
```

Comment:
Small point, wondering if this line is valuable. Probs won't restart prometheus pods unless there's an update. Maybe we should update the readme to do the make instead? What do you think?

Reply:
This command will be used when istio has been upgraded, in which case I think this line will cause the prometheus pod to restart due to `istioctl kube-inject`, which is what we want - is that right?

Reply:
Yep, that's true. 👍